a.js

```diff
@@ -0,0 +1,9 @@
+const humberger = document.querySelector('.humberger');
+const ulnav = document.querySelector('.ulnav');
```

Comment:
- [ ] [_OPTIONAL_] It's better to give meaningful file names, instead of a.js better to make it main.js or index.js.